lib/net/pool.js

```diff
@@ -704,6 +729,60 @@ class Pool extends EventEmitter {
     this.compactBlocks.clear();
   }
 
+  /**
+   * Start the filters headers sync.
+   */
+
+  async startFilterHeadersSync() {
+    this.filterSyncing = true;
+    this.logger.info('Starting filter headers sync (%s).',
+      this.chain.options.network);
+    if (!this.opened || !this.connected)
+      return;
+
+    const cFHeaderHeight = this.cfHeaderChain.tail.height;
+    const startHeight = cFHeaderHeight
+    ? cFHeaderHeight + 1 : 1;
+    const chainHeight = await this.chain.height;
+    const stopHeight = chainHeight - startHeight + 1 > 2000
+     ? startHeight + 1999 : chainHeight;
+    const stopHash = await this.chain.getHash(stopHeight);
+    this.requestedFilterType = common.FILTERS.BASIC;
+    this.requestedStopHash = stopHash;
+    await this.peers.load.sendGetCFHeaders(
```

Comment:
two thoughts here
1. i don't think this needs to be exclusive to the loader peer
2. this may get called before there are ANY peers connected
  - It should handle that case graefully (like use for each peer logic so if there are no peers, nothing happens)
  - the function should ALSO be called from (for example maybe) `handleOpen()` just like how we call `sendSync()`. It maybe even could be combined with `sendSync()` if neutirno? i dunno whats best

Reply:
I think everytime we add a new block header after IBD we should ask for filters from there too